From 382dd6926ab087de507c054a15dd30295637f773 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 19 Oct 2024 07:59:54 +0000
Subject: [PATCH] Set active state to false on drop

The 'leave' event is triggered on Xorg only when the pointer leaves the drop target, but not when the drop event happened.
---
 src/window.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/window.py b/src/window.py
index 4cd9598..6b32ecf 100644
--- a/src/window.py
+++ b/src/window.py
@@ -229,6 +229,7 @@ class LetterpressWindow(Adw.ApplicationWindow):
         return Gdk.DragAction.COPY
 
     def __on_drop(self, widget, drop, *args):
+        self.drag_revealer.set_reveal_child(False)
         failed_as_file = False
         try:
             self.check_is_image(drop)
-- 
GitLab

